core/editor/editor.php

```diff
@@ -588,10 +592,7 @@ public function enqueue_scripts() {
 				'darkModeStylesheetURL' => ELEMENTOR_ASSETS_URL . 'css/editor-dark-mode' . $suffix . '.css',
 				'defaultGenericFonts' => $kits_manager->get_current_settings( 'default_generic_fonts' ),
 			],
-			// Legacy Mode - for backwards compatibility of older HTML markup.
```

Comment:
Where is the BC?

Reply:
https://github.com/elementor/elementor/blob/release/3.1.0/assets/dev/js/editor/editor-base.js#L789